src/lua/LuaEvent.h

```diff
@@ -51,7 +51,6 @@ namespace LuaEvent {
 	{
 		Queue(GetEventQueue(), event);
 	}
-
```

Comment:
Errant deleted whitespace here - preferably LuaEvent isn't touched at all by this PR.

Reply:
PR isn't cleaned up yet.. but thanks for pointing it out.